lightning/src/chain/mod.rs

```diff
@@ -175,6 +175,67 @@ pub trait Confirm {
 	fn get_relevant_txids(&self) -> Vec<Txid>;
 }
 
+/// An error enum representing a failure to persist a channel monitor update.
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum ChannelMonitorUpdateErr {
```

Comment:
Hmm... it's a little strange this error is related to persisting `ChannelMonitor` updates but lives in a different location than both `Persist` and `ChannelMonitor`. Not sure if there is a compelling case for moving this and `Persist`. At very least I'd think this should live next to `Persist`, wherever that would be. `chain::Watch` is just forwarding these from a `Persist` implementation.